ui/src/components/WelcomeTutorial.tsx

```diff
@@ -0,0 +1,431 @@
+    padding: 0px;
+    left: 455px;
+    top: 29px;
+    width: 14px;
+    height: 14px;
+
+  && {
+    box-shadow: none;
+  }
+  }
+`;
+
+const StyledWelcomeWelcome = styled(Dialog)`
+  .MuiDialog-paper {
+    width: 490px;
+    height: 346px;
+    background: #ffffff;
+    box-shadow: 0px 0px 25px 5px rgba(0, 0, 0, 0.1);
+    border-radius: 10px;
+    margin: 0px;
+  }
+`;
+
+const StyledWelcomeButton = styled(Button)`
+  position: absolute;
+  left: 134px;
+  bottom: 24px;
+  .MuiButton-label {
+    color: ${colors.primaryDark2};
+    font-size: 16px;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 150%;
+    margin: 0px 10px;
+  }
+  && {
+    box-shadow: none;
+    background-color: ${colors.primaryLight4};
+    width: 222px;
+    height: 48px;
+    display: flex;
+    &:active {
+      background-color: ${colors.primaryLight3};
+    }
+  }
+`;
+
+const StyledWelcomeLogo = styled.img`
+  margin-left: 189px;
+  margin-right: 196px;
+  margin-bottom: 16px;
+  display: flex;
+  background-repeat: no-repeat;
+  width: 105px;
+  height: 109px;
+`;
+
+const StyledWelcomeCloseIcon = styled(CloseIcon)`
+  .MuiIcon-colorPrimary {
+    color: ${colors.primaryDark2};
+  }
+`;
+
+const StyledContainer = styled.div`
+  display: inline-block;
+`;
+
+const StyledNavigateIconButton = styled(IconButton)`
+    position: absolute;
+    margin: 0px;
+    padding: 0px;
+    left: 372px;
+    top: 29px;
+    width: 14px;
+    height: 14px;
+
+  && {
+    box-shadow: none;
+  }
+  }
+`;
+
+const StyledNavigateWelcome = styled(Dialog)`
+  .MuiDialog-paper {
+    width: 415px;
+    height: 230px;
+    background: #ffffff;
+    box-shadow: 0px 0px 25px 5px rgba(0, 0, 0, 0.1);
+    border-radius: 10px;
+    margin: 0px;
+  }
+`;
+
+const StyledNavigateNextButton = styled(Button)`
+  left: 229px;
+  .MuiButton-label {
+    color: ${colors.primaryDark2};
+    font-size: 16px;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 150%;
+    margin: 0px 10px;
+  }
+  && {
+    box-shadow: none;
+    background-color: ${colors.primaryLight4};
+    width: 80px;
+    height: 40px;
+    &:active {
+      background-color: ${colors.primaryLight3};
+    }
+  }
+`;
+
+const StyledNavigateCloseIcon = styled(CloseIcon)`
+  .MuiIcon-colorPrimary {
+    color: ${colors.primaryDark2};
+  }
+`;
+
+const StyledNavigateTag = styled(Button)`
+  width: 58px;
+  height: 32px;
+  left: 24px;
+  border-radius: 5px;
+  background-color: ${colors.grey};
+
+  .MuiButton-label {
+    color: ${colors.neutralDark};
+    font-size: 16px;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 24px;
+    margin: 0px 10px;
+  }
+  && {
+    box-shadow: none;
+    background-color: ${colors.grey};    
+`;
+
+const StyledNavigateContainer = styled.div`
+  display: inline-block;
+`;
+
+export function WelcomeTutorial() {
+  const [open, setOpen] = React.useState("first");
```

Comment:
Let's move these strings, `first`, `second`, into a string enum.

That way we can refer to it as `Tutorial.First`, `Tutorial.Second` etc.

Reply:
Hmm, also thinking about this more, should the tutorial be handling it's own state? Maybe we should lift it up to the parent component

Reply:
The component should be able to handle its own state to go from screen to screen right? I added it as a prop now too so that it can be set from the parent component as well (to launch or close) the tutorial

Reply:
Yeah you're right, I think this works for now.